readme.md

```diff
@@ -1023,6 +1023,21 @@ echo $faker->vat;           // "A35864370"
 echo $faker->dni; // '83367512'
 ```
 
+### `Faker\Provider\fa_IR\Person`
+
+```php
+<?php
+
+// Generates a random female first name
+echo $faker->firstNameFemale; // "مارال"
+
+// Generates a random last name
+echo $faker->lastName; // "کامکار"
```

Comment:
please don't document formatters that exist on all locales, only the ones that are specific to your locale

Reply:
You got it.